x-pack/plugins/upgrade_assistant/public/application/components/overview/fix_issues_step/fix_issues_step.tsx

```diff
@@ -49,10 +51,45 @@ const FixIssuesStep: FunctionComponent<Props> = ({ setIsComplete }) => {
   );
 };
 
+interface CustomProps {
+  navigateToEsDeprecationLogs: () => void;
+}
+
+const AccessDeprecationLogsMessage = ({ navigateToEsDeprecationLogs }: CustomProps) => {
+  return (
+    <WithPrivileges privileges={`index.${DEPRECATION_LOGS_INDEX}`}>
+      {({ hasPrivileges, isLoading }) => {
+        if (isLoading || !hasPrivileges) {
+          // Don't show the message with the link to access deprecation logs
+          // to users who can't access the UI anyways.
+          return null;
+        }
+
+        return (
+          <FormattedMessage
+            id="xpack.upgradeAssistant.overview.accessEsDeprecationLogsLabel"
+            defaultMessage="Optionally you can consult the {esDeprecationLogsLink} to make sure your application is not using any deprecated APIs."
```

Comment:
This isn't an optional part of the upgrade process if you have application code that calls ES APIs. Maybe:

```suggestion
            defaultMessage="If you have application code that calls Elasticsearch APIs, review the {esDeprecationLogsLink} to make sure you are not using deprecated APIs."
```

Reply:
👍